packages/replayio/src/utils/recordings/types.ts

```diff
@@ -13,19 +13,23 @@ export enum RECORDING_LOG_KIND {
   writeStarted = "writeStarted",
 }
 
-// This data comes from the runtime
+export type UnstructuredMetadata = Record<string, unknown>;
+
+// This data primarily comes from the runtime
+// The CLI adds some entries as well, based on upload status
 export type LogEntry = {
   buildId?: string;
   data?: any;
   driverVersion?: string;
   id: string;
   kind: RECORDING_LOG_KIND;
-  metadata?: {
+  metadata?: Object & {
```

Comment:
`Object` type is *rarely* a good fit - it corresponds to the actual `Object`'s prototype. I've removed this locally to see where the CLI wouldn't compile but everything is OK without it so maybe we can just remove this?

Reply:
An index signature is appropriate:
```ts
  metadata?: {
    argv?: string[];
    uri?: string;
    [k: string]: unknown;
  };
```

Reply:
Ah, right. Nice. Thanks!